activestorage/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   `TransformJob` allows previewables being `preprocessed` when an attachment has predefined variants.
```

Comment:
Readers probably won't be aware of `TransformJob`, so let's describe this just in terms of `preprocessed: true`.

```suggestion
*   Fix `preprocessed: true` option for named variants of previewable files.
```